application_form/services/queue.py

```diff
@@ -144,7 +147,10 @@ def _calculate_queue_position(
     ).order_by("queue_position")
```

Comment:
(GitHub doesn't seem to let me comment on line 144, which has the `only` expression so commenting to this line instead.)

Shouldn't the is-old-batch field be also added to the fields listed in `only`?